cluster/gce/config-default.sh

```diff
@@ -550,12 +545,12 @@ export TLS_CIPHER_SUITES=""
 
 # CLOUD_PROVIDER_FLAG defines the cloud-provider value presented to KCM, apiserver,
 # and kubelet
-export CLOUD_PROVIDER_FLAG="${CLOUD_PROVIDER_FLAG:-gce}"
+export CLOUD_PROVIDER_FLAG="${CLOUD_PROVIDER_FLAG:-external}"
 
 # When ENABLE_AUTH_PROVIDER_GCP is set, following flags for out-of-tree credential provider for GCP
 # are presented to kubelet:
 # --image-credential-provider-config=${path-to-config}
 # --image-credential-provider-bin-dir=${path-to-auth-provider-binary}
 # Also, it is required that DisableKubeletCloudCredentialProviders and KubeletCredentialProviders
 # feature gates are set to true for kubelet to use external credential provider.
-ENABLE_AUTH_PROVIDER_GCP="${ENABLE_AUTH_PROVIDER_GCP:-false}"
```

Comment:
should I modify this feature gates mentioned in the comment too?

`DisableKubeletCloudCredentialProviders and KubeletCredentialProviders`

Reply:
I guess I would have expected ENABLE_AUTH_PROVIDER_GCP=true to drive setting those feature gates?

Reply:
ok, one seems to be GA
```
kg/credentialprovider/plugins.go:      return utilfeature.DefaultFeatureGate.Enabled(features.DisableKubeletCloudCredentialProviders)
pkg/features/kube_features.go:  DisableKubeletCloudCredentialProviders featuregate.Feature = "DisableKubeletCloudCredentialProviders"
pkg/features/kube_features.go:  DisableKubeletCloudCredentialProviders: {Default: false, PreRelease: featuregate.Alpha},
aojea@aojea:~/src/kubernetes$ grep -r KubeletCredentialProviders pkg/
pkg/features/kube_features.go:  KubeletCredentialProviders featuregate.Feature = "KubeletCredentialProviders"
pkg/features/kube_features.go:  KubeletCredentialProviders: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.28
```